owslib/ogcapi/processes.py

```diff
@@ -65,29 +68,31 @@ def job_list(self, process_id: str) -> dict:
         path = f'processes/{process_id}/jobs'
         return self._request(path)
 
-    def execute(self, process_id: str, json: dict) -> dict:
+    def execute(self, process_id: str, json: dict) -> str:
         """
         implements: POST /processes/{process-id}/jobs
 
         Executes a process, i.e. creates a new job. Inputs and outputs will have
         to be specified in a JSON document that needs to be send in the POST body.
 
-        @returns: `dict` of ...
+        @returns: `str` of the status location
         """
 
         path = f'processes/{process_id}/jobs'
-        return self._request_post(path, json)
+        resp = self._request_post(path, json)
+        data = resp.json()
+        return resp.headers.get("Location", data["location"])
 
-    def _request_post(self, path: str, json: dict) -> dict:
+    def _request_post(self, path: str, json: dict) -> requests.Response:
```

Comment:
I would expect a json response

Reply:
In this case, we actually need the response object as the OGC-API specifies that the job Execution should return the Status endpoint via `Location` header. I would also prefer the JSON object, but technically we need more than the body here.

Reply:
We can make our own dict which contains all relevant information. But it would be handled in `owslib`.